app/models/search/tag_search_form.rb

```diff
@@ -30,4 +35,29 @@ def search_results
     @searcher.search_results
   end
 
+  def set_fandoms
+    return if @options[:fandoms].blank?
+
+    names = @options[:fandoms].split(",").map(&:squish)
+    @options[:fandom_ids] = Tag.where(name: names).pluck(:id)
+  end
+
+  def sort_columns
+    options[:sort_column] || "name"
+  end
+
+  def sort_direction
+    options[:sort_direction] || default_sort_direction
+  end
+
+  def sort_options
+    [
+      %w[Name name],
+      %w[Date\ Created created_at],
```

Comment:
You can use `["Date Created", "created_at"]` here. Rubocop won't ask for a switch to `%w` if one of the strings has a space in it.

The other line can stay `%w[Name name],` though.